model/src/w3iorsmd.F90

```diff
@@ -619,7 +621,11 @@ SUBROUTINE W3IORS ( INXOUT, NDSR, DUMFPI, IMOD, FLRSTRT )
     !          ( Bail out if write for TYPE.EQ.'WIND' )
     !
     IF ( WRITE ) THEN
-      IF ( TYPE.EQ.'WIND' .OR. TYPE.EQ.'CALM' ) THEN
+      IF ( TYPE.EQ.'FTCH' ) THEN
+        RPOS  = 1_8 + LRECL*(3-1_8)
+        WRITE (NDSR,POS=RPOS,ERR=803,IOSTAT=IERR) FETCH
```

Comment:
@MHBalsmeier    After a recent PR we need to update this to instead be: 

          WRITE (NDSR,POS=RPOS,IOSTAT=IERR) FETCH
            IF (IERR.NE.0) CALL EXTIOF(NDSE,IERR,'W3IORS','',31, &
                                       ISWRITE=.TRUE.,POS=RPOS)